elastic/kibana
# [Cases] Change value type of text custom fields to string.

## Summary

The `value` field for custom fields of type `CustomFieldType.TEXT` is now `string`(was previously `string[]`).

I changed the UI and the backend, please double-check.

Testing the UI:
1. Created two custom fields of type `text`. One required one not required.
2. Create Case Form:
    - Populate both fields
    - Populate only the required field
3. Case Detail View
    - Edit the required text field
    - Edit the not-required text field
    - User actions should display correctly
    - Text box should show correctly
    - Text should show correctly(on not editing mode)
    - Deleting the text from the field should work correctly
